src/giddyup.erl

```diff
@@ -27,6 +27,7 @@
 get_suite(Platform) ->
```

Comment:
@jburwell Currently the logs are *only* copied locally for a non-GiddyUp run.  If we'd like to change that, we can.  We'd also have to make sure that local logs are then periodically cleaned up the builder.  Or just decide that once the files are uploaded we simply get rid of them.